tests/e2e/modules/weather_spec.js

```diff
@@ -43,7 +43,7 @@ describe("Weather module", function () {
 				const weather = generateWeather();
 				await setup({ template, data: weather });
 
-				return app.client.waitUntilTextExists(".weather .normal.medium span:nth-child(2)", "6 WSW", 10000);
+				return await app.client.$(".weather .normal.medium span:nth-child(2)", "6 WSW", 10000);
```

Comment:
You should be able to return the promise before awaiting on it, like you did in the other test files.

e.g.

```js
it("Show happy new year compliment on new years day", async function () {
  const elem = await app.client.$(".compliments");
  return elem.getText(".compliments").then(function (text) {
    expect(text).to.be.oneOf(["Happy new year!"]);
  });
});
```